api/client/network.go

```diff
@@ -113,6 +113,7 @@ func (cli *DockerCli) CmdNetworkRm(args ...string) error {
 			status = 1
 			continue
 		}
+		fmt.Fprintf(cli.out, "%s\n", net)
```

Comment:
Yeah, was thinking to change that for those places where it isn't needed. It's nit-picking probably, so don't let it block this PR 👍